cmd/pj-rehearse/server.go

```diff
@@ -250,7 +250,7 @@ func (s *server) handleNewPush(l *logrus.Entry, event github.PullRequestEvent) {
 			jobTableLines = append(jobTableLines, fmt.Sprintf("A full list of affected jobs can be found [here](%s%s)", s.rehearsalConfig.GCSBrowserPrefix, fileLocation))
 		}
 		jobTableLines = append(jobTableLines, s.getUsageDetailsLines()...)
-		if err = s.ghc.CreateComment(org, repo, number, strings.Join(jobTableLines, "\n")); err != nil {
+		if err := s.ghc.CreateComment(org, repo, number, strings.Join(jobTableLines, "\n")); err != nil {
```

Comment:
Making it look like you're handling the error produced by CreateComment but implicitly shadowing the error from the larger scope is super pernicious